langs/pt.json

```
@@ -0,0 +1,189 @@
+{
+  "locale": "en",
```

it's failing because you are overwriting the en locale instead of pt
```suggestion
  "locale": "pt",
```